x/ophost/types/output.go

```diff
@@ -28,19 +28,24 @@ func GenerateOutputRoot(version byte, storageRoot []byte, latestBlockHash []byte
 }
 
 func GenerateWithdrawalHash(bridgeId uint64, l2Sequence uint64, sender string, receiver string, denom string, amount uint64) [32]byte {
```

Comment:
we might delete spliter if we use 32 bytes hash digests as variables

Reply:
yea seems we can remove it applied